support/config/deployment.go

```diff
@@ -358,7 +374,12 @@ func (c *DeploymentConfig) setLocation(hcp *hyperv1.HostedControlPlane, multiZon
 	c.setColocation(hcp)
 	// TODO (alberto): pass labels with deployment hash and set this unconditionally so we don't skew setup.
 	if c.Replicas > 1 {
-		c.setMultizoneSpread(multiZoneSpreadLabels)
+		multiZoneRequired := true
+		// On OpenStack, we can't guarantee that the cloud is configured with multiple availability zones.
+		if hcp.Spec.Platform.Type == hyperv1.OpenStackPlatform {
+			multiZoneRequired = false
+		}
```

Comment:
Let's add KubeVirt to this as well.

The reality is that the required zone spread has caused us (kubevirt platform) more trouble than it has benefited people. I think setting preferred during scheduling for zone labeling is a happy medium for us as well.